EUR-Lex.js

```diff
@@ -37,42 +37,74 @@
 
 
 // attr()/text() v2
-function attr(docOrElem,selector,attr,index){var elem=index?docOrElem.querySelectorAll(selector).item(index):docOrElem.querySelector(selector);return elem?elem.getAttribute(attr):null;}function text(docOrElem,selector,index){var elem=index?docOrElem.querySelectorAll(selector).item(index):docOrElem.querySelector(selector);return elem?elem.textContent:null;}
+function attr(docOrElem, selector, attr, index) {
+	var elem = index ? docOrElem.querySelectorAll(selector).item(index) : docOrElem.querySelector(selector); return elem ? elem.getAttribute(attr) : null;
+}
```

Comment:
No, this can stay minimized as it was before. Just add the comment `// eslint-disable-next-line` before it, see also https://github.com/zuphilip/translators/wiki/Common-code-blocks-for-translators#replacements-for-xpath